Engineering/CodeReviews/Python.md

```diff
@@ -24,5 +24,13 @@ We follow the [PEP 257](https://www.python.org/dev/peps/pep-0257/) Docstring con
 
 Flake8 is an option for teams who want a single wrapper for running Pyflakes and pycodestyle.
 
+## Secure Coding
```

Comment:
When adding a tool, I always worry about conflicts, though I'd hope a security tool and a linter wouldn't conflict. Have you used Bandit and the other recommended tools all using the default configurations? Is there any guidance we should have around configuring Bandit to work without conflicts?

Obviously the other tools mentioned here don't have guidance, but I think they need it, and we need to be prescriptive on what combination of tools has been successful.

Reply:
This is a good point, there isn't a conflict at least from how I've used it in the sense that the linting happens on the fly, continuously in VSCode while Bandit is run after explicitly. That said, I didn't have any specific guidance in mind, the profiles to be used (or you can run all) depend on a prelim analysis of where you think the code that you write might be vulnerable but also an awareness of the profiles that Bandit has to offer. 

That said, as you pointed, given that at the moment we don't actually provide any guidance on the use of the other tools, we might want to identify a template on doing so such that we have consistency in providing a recommended approach for any tool that is shared. Thoughts?